Frontend/CO.CDP.OrganisationApp/Pages/Consortium/ConsortiumChangeEmail.cshtml

```diff
@@ -0,0 +1,46 @@
+@page "/consortium/{id}/change-email"
+@model ConsortiumChangeEmailModel
+@using CO.CDP.Localization
+
+@{
+    ViewData["Title"] = StaticTextResource.Consortium_ConsortiumChangeEmail_Heading;
+    var emailAddressHasError = ((TagBuilder)Html.ValidationMessageFor(m => m.EmailAddress)).HasInnerHtml;
+    var backLink = $"/consortium/{Model.Id}/overview";
+}
+
+@section BeforeContent {
+        <a href="@backLink" class="govuk-back-link">@StaticTextResource.Global_Back</a>
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <partial name="_ErrorSummary" model="@Model.ModelState" />
+        <form class="form" method="post" novalidate>
+            @if (Request.Query.ContainsKey("frm-summary"))
+            {
+                <input type="hidden" name="RedirectToSummary" value="true" />
+            }
+            <span class="govuk-caption-l">
+                @Model.ConsortiumName
+            </span>
```

Comment:
Caption elements should go inside the h1 to form part of the main heading

Reply:
Done